### PR TITLE
docs(jans-cedarling): update schema

### LINKED ISSUE #10491: docs(jans-cedarling): update schema
-------------------

### Description

As Agama Lab has been fixed, we need to revert https://github.com/JanssenProject/jans/pull/10395 and additionally adjust the HTTP Request schema:
```
entity HTTP_Request = {
  "header": {
    "Accept"?: String, 
    }, 
    "url": Url
};
```

-------------------

### Prepare

- [x] Read contribution guidelines
- [x] Read license information

------------------

### Identified code changes

N/A

-------------------

### Test cases and code coverage

N/A

-------------------

### Document the changes

- [ ] Update wiki

### PR DESCRIPTION
**Pull request by SafinWasi**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #10491 

#### Implementation Details
  

-------------------
### Test and Document the changes
-N/A

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [ ] **I confirm that there is no impact on the docs due to the code changes in this PR.**

### CODE REVIEW

File: jans-cedarling/schema/cedarling_core.cedarschema [hunk 1]
```diff
@@ -90,42 +91,42 @@ namespace Jans {
 
     // ******  Actions  ******
     action Compare appliesTo {
-      principal: [User, Workload, Role],
+      principal: [User, Workload],
```
Comment (nynymike) [inline jans-cedarling/schema/cedarling_core.cedarschema:94]: Why did you remove role for all these?

**Review by nynymike — CHANGES_REQUESTED** [review]
I don't understand why you removed `Role` for the actions.

File: jans-cedarling/schema/cedarling_core.cedarschema (hunk 1 above)
Comment (SafinWasi) [reply jans-cedarling/schema/cedarling_core.cedarschema:94]: The role was added as a workaround because Agama lab could not create RBAC policies without Role being defined in actions, even if this was legal in cedar. This bug has since been fixed.

**Review by olehbozhok — APPROVED** [review]

**Review by nynymike — APPROVED** [review]
